src/main/java/seedu/duke/helper/UI.java

```diff
@@ -33,96 +33,81 @@ public void printNewLineSeparator() {
+                + "add patient /info T4867591Z, Mary Douglas Owen, 25, F, 15 King's Avenue, 1997-08-26, 2020-03-30");
     }
 
     public void printAddDoctorExampleMessage() {
-        System.out.println("Please note the error(s) mentioned above and try again!");
-        System.out.println("Here are two examples. Please follow the input order:");
-        System.out.println("add doctor /info S1234567A, John Doe, 23, M, 10 Baker Street,"
-                + " 1999-12-31, Urinology");
-        System.out.println("add doctor /info T4867591Z, Mary Douglas Owen, 25, F, 15 King's Avenue,"
-                + "1997-08-26, Optometry");
+        printParagraph("Please note the error(s) mentioned above and try again!\n"
+                + "Here are two examples. Please follow the input order:\n"
+                + "add doctor /info S1234567A, John Doe, 23, M, 10 Baker Street, 1999-12-31, Urinology\n"
+                + "add doctor /info T4867591Z, Mary Douglas Owen, 25, F, 15 King's Avenue, 1997-08-26, Optometry");
     }
 
     public void printDeletePatientExampleMessage(PatientList patientList) {
-        System.out.println("Please input a positive number up to " + patientList.getSize() + " only.");
-        System.out.println("Here is an example:");
-        System.out.println("delete patient /info 1");
+        printParagraph("Please input a positive number up to " + patientList.getSize() + " only.\n"
+                + "Here is an example:\ndelete patient /info 1");
     }
 
     public void printDeleteDoctorErrorMessage(DoctorList doctorList) {
-        System.out.println("Please input a positive number up to " + doctorList.getSizeDoctor() + " only.");
-        System.out.println("Here is an example:");
-        System.out.println("delete doctor /info 1");
+        printParagraph("Please input a positive number up to " + doctorList.getSizeDoctor() + " only.\n"
+                + "Here is an example:\ndelete doctor /info 1");
     }
 
     public void printNullParametersMessage() {
-        System.out.println("Parameters missing or not detected. Please use /info for parameters.");
+        printParagraph("Parameters missing or not detected. Please use /info for parameters.");
     }
 
-    public void printPrompt() {
+    public static void printPrompt() {
         System.out.print("You: ");
     }
 
-    public void printParagraph(String paragraph) {
-        System.out.print("HalpMi: ");
+    public static void printParagraph(String paragraph) {
         String[] arrayOfSentences = paragraph.split("\n");
-        for (String s : arrayOfSentences) {
+        System.out.println("HalpMi: " + arrayOfSentences[0]);
+        for (int i = 1; i < arrayOfSentences.length; i++) {
             System.out.print("        ");
-            System.out.println(s);
+            System.out.println(arrayOfSentences[i]);
         }
     }
 
-    public void printCont(String sentence) {
+    public static void printCont(String sentence) {
```

Comment:
Cont means Continue, right? Or is there a better name?

Reply:
Yes it means Continue